experimental/packages/opentelemetry-instrumentation/README.md

```diff
@@ -219,12 +219,58 @@ If nothing is specified the global registered provider is used. Usually this is
 There might be use case where someone has the need for more providers within an application. Please note that special care must be takes in such setups
 to avoid leaking information from one provider to the other because there are a lot places where e.g. the global `ContextManager` or `Propagator` is used.
 
-## Instrumentation for ES Modules In Node.js (experimental)
+## Instrumentation for ECMAScript Modules (ESM) in Node.js (experimental)
 
-As the module loading mechanism for ESM is different than CJS, you need to select a custom loader so instrumentation can load hook on the ESM module it want to patch. To do so, you must provide the `--experimental-loader=@opentelemetry/instrumentation/hook.mjs` flag to the `node` binary. Alternatively you can set the `NODE_OPTIONS` environment variable to `NODE_OPTIONS="--experimental-loader=@opentelemetry/instrumentation/hook.mjs"`.
-As the ESM module loader from Node.js is experimental, so is our support for it. Feel free to provide feedback or report issues about it.
+Node.js uses a different module loader for ECMAScript Modules (ESM) vs. CommonJS (CJS).
+A `require()` statement will cause Node.js to use the CommonJS module loader.
+An `import()` statement will cause Node.js to use the ECMAScript module loader.
 
-**Note**: ESM Instrumentation is not yet supported for Node 20.
+### TypeScript
+
+Many TypeScript projects today are written using ESM syntax, regardless of how they are compiled.
+In the `tsconfig.json`, there is an option to compile to ESM or CJS.
+If the compiled code is ESM, those import statements will remain the same (e.g. `import { foo } from 'bar';`).
+If the compiled code is CJS, those import statements will become `require()` statements (e.g. `const { foo } = require('bar');`)
+
+For more explanation about CJS and ESM, see the [Node.js docs](https://nodejs.org/dist/latest-v18.x/docs/api/modules.html#enabling).
+
+### Initializing the SDK
```

Comment:
I'm finding it hard to review because there are a number of cross-cutting things here. :weak-smile:
I'll add a comment to the parent #4845 explaining.

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/4845#issuecomment-2253556217

Reply:
I ended up adding a separate doc here with all the details, then linked to it from various READMEs. Let me know what you think about this approach. My thought is to have it in one place here for easy updating, then use that as a reference where needed to avoid duplicating content.